src/sites_toolbox.py

```diff
@@ -152,18 +164,48 @@ def sites_toolbox_page():
         st.write("Here are the site details for site", site_id, ":", site_details)
         if st.button("Close site details"):
             st.empty()
-    
+
     # getting site group details
     st.markdown(
         f'<h1 style="color:#63BCAF;font-size:32px;">{"Get Site Group Details"}</h1>',
         unsafe_allow_html=True,
     )
     site_group_name = st.selectbox("Enter the site group name.", site_groups)
     if st.button("Get site group details"):
-        site_group_sites, site_group_users = get_site_group_details(session=session, site_group_name=site_group_name)
-        st.write("Site group", site_group_name, "contains the following", len(site_group_sites), "sites:", site_group_sites)
-        st.write("The following", len(site_group_users), "users are part of this group:", site_group_users)
+        site_group_sites, site_group_users = get_site_group_details(
+            session=session, site_group_name=site_group_name
+        )
+        st.write(
+            "Site group",
+            site_group_name,
+            "contains the following",
+            len(site_group_sites),
+            "sites:",
+            site_group_sites,
+        )
+        st.write(
+            "The following",
+            len(site_group_users),
+            "users are part of this group:",
+            site_group_users,
+        )
         if st.button("Close site group details"):
             st.empty()
 
-  
+    # update user site group
+    st.markdown(
+        f'<h1 style="color:#63BCAF;font-size:32px;">{"Change User Site Group"}</h1>',
+        unsafe_allow_html=True,
+    )
+    email = st.selectbox("Select user whose site group will change.", user_list)
+    site_group_name = st.selectbox(
+        "Select the site group the user will join.", site_groups
+    )
+
+    if st.button("Change user's site group"):
+        user, user_site_group = change_user_site_group(
+            session=session, email=email, site_group_name=site_group_name
+        )
+        st.write(user, "is now in the", user_site_group, "site group.")
```

Comment:
i like to using f strings.

f"{user} is now in the {user_site_group} site group"